README.md

```diff
@@ -97,4 +196,34 @@ There is a Membership Attributes Service radiator. This uses standard ELB and Dy
 
 The packer cloud formation template should be used to create an AMI with Oracle Java 8 installed. The base AMI which should be used with this is Ubuntu Trusty 14.04 (ami-acc41cdb).
 
-The output AMI from packer should be used with the membership-attribute-service cloud formation template. 
+The output AMI from packer should be used with the membership-attribute-service cloud formation template.
+ 
+## Testing SalesForce hook
+ 
+In order to trigger the SaleForce hook in `DEV`, you have to generate a `POST` request to the SalesForce's endpoint in Members Data API. 
```

Comment:
A couple of small typos in this line:

"In order to trigger the Sale**sf**orce hook..."
"...request to the Salesforce ~~'s~~ endpoint..."